guides/gatherers.md

```diff
@@ -242,6 +243,43 @@ Example output (in Rhai):
 #{ nodeid: 2, ring0_addr: "10.80.1.12" };
 ```
 
+### dir_scan
+
+**Argument required**: Yes, a glob pattern for directory/files scanning
+
+Example argument:
```

Comment:
Let's put `Example arguments` after the main description (`This gatherer allows...`).

Something like:
```
...information associated to each file.

The provided argument is a glob pattern for directory/files scanning.

Example arguments:
- `/usr/sap/[A-Z][A-Z0-9][A-Z0-9]/ERS[0-9][0-9]`
- `/etc/polkit-1/rules.d/[0-9][0-9]-SAP[A-Z][A-Z0-9][A-Z0-9]-[0-9][0-9].rules`
```